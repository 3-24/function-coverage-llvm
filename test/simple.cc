#include<stdio.h>

void a(){
}

void b(){
}

void c(){
}

int main(int argc, char *argv[]) {
    for (int i=1; i<argc; i++){
        if (argv[i][0] == 'a'){
            a();
        } else if (argv[i][0] == 'b'){
            b();
        } else if (argv[i][0] == 'c'){
            c();
        }
    }

    return 0;
}