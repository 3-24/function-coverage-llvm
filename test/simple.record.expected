0 main at /function-sequence-llvm/test/simple.cc
1 a() at /function-sequence-llvm/test/simple.cc
2 b() at /function-sequence-llvm/test/simple.cc
3 c() at /function-sequence-llvm/test/simple.cc
